addons/l10n_bd/data/template/account.account-bd.csv

```diff
@@ -0,0 +1,138 @@
+"id","name","reconcile","tag_ids","code","account_type"
+"l10n_bd_100101","Liquidity Transfer","True","","100101","asset_current"
+"l10n_bd_100102","Bank Suspense Account","True","","100102","asset_current"
+"l10n_bd_100103","Outstanding Receipts","True","","100103","asset_current"
+"l10n_bd_100104","Outstanding Payments","True","","100104","asset_current"
+"l10n_bd_100105","Bank","False","","100105","asset_cash"
```

Comment:
Are these needed to be defined by hand instead of letting them being created automatically?
It's interesting to do so if we want to define explicit accounts/code for them. I don't know if that's the case here.
Some of them are duplicated because you don't provide them in the `template_bd.py`.
Example `Liquidity Transfer` is created with this prefix https://github.com/odoo/odoo/pull/160421/files#diff-bde73121693a965dae8acedebb3be0bad1180f202d81042b2b4f1bd72aa66ccfR25 (btw, it seems like a weird code prefix to place it). So it is doubled
You can see them automatically defined them here https://github.com/odoo/odoo/blob/17.0/addons/account/models/chart_template.py#L672 (if you don't define the account with the right key in the template data)

Reply:
So, We can actually rely on the `template_bd.py` in order to generate those accounts instead of having them in the `account.csv` file. If yes, Could you please tell me why those accounts specially?

Reply:
Yes.
You can find them in the method I linked above https://github.com/odoo/odoo/blob/17.0/addons/account/models/chart_template.py#L672
You can see that we generate a dict with the accounts you wrote there.
You see here https://github.com/odoo/odoo/blob/17.0/addons/account/models/chart_template.py#L738 that we check the field in `company`. This one already got data based on your method `_get_bd_res_company` in `template_bd.py`.
If the field is already present in the `company`, we remove the default.

So, you could generate everything by hand in the csv, but then you have to link them all in the method `_get_bd_res_company`, to not create them twice.

Here https://github.com/odoo/odoo/blob/17.0/addons/account/models/chart_template.py#L588, you can see that we have something for the unaffected_earnings account too.
(You can also see the rest of the setup with the accounts on the journals, the taxes, other properties, ...)

Reply:
I did link them based on the PO requirement in the card chatter.